apps/designer/app/designer/features/sidebar-left/sidebar-left.tsx

```diff
@@ -91,7 +73,10 @@ export const SidebarLeft = ({
             ...sidebarTabsContentStyle,
             // We need the node to be rendered but hidden
             // to keep receiving the drag events.
-            visibility: isDragging ? "hidden" : "visible",
+            visibility:
+              dragAndDropState.isDragging && dragAndDropState.origin === "panel"
```

Comment:
is that parent getting isDragging true when origin is not panel? Just wondering

Reply:
Not sure what parent you're referring to, but basically we need the visibility hack only when the drag is started on the panel. We could apply the hack always though, wouldn't hurt.

Reply:
what I mean is, do we get notified when the drag started not on the panel? I mean we are inside the panel, that's why I am wondering, why do we know about the drag started outside the panel, just a question

Reply:
Sure, the dragAndDropState gets updated no matter where the drag has started